students/JimJenkins/session07/html_render2

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+"""
+author: Jim Jenkins (dvlupr)
+Date: 12/10/2018
+
+Note: I didn't write this the way the original assignment focuses, rather it was written for some level of 
+understanding in OOP.
+"""
```

Comment:
the assignment is designed to cover certain aspects of OO subclassing, etc.

I can't really see where you're going here...